ndk/lib/src/androidTest/java/io/sentry/ndk/sample/NdkSample.java

```diff
@@ -0,0 +1,13 @@
+package io.sentry.ndk.sample;
+
+public class NdkSample {
+  static {
+    System.loadLibrary("sentry-android-test");
+  }
+
+  public static native void crash();
```

Comment:
Right now we don't have tests to test `crash()` or `message()`. I'd either remove them or add tests, similar to the blackbox text for `transaction()`